recipes/festlex-cmu/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "festlex-cmu" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://festvox.org/packed/festival/2.5/festlex_CMU.tar.gz
+  sha256: c19430919bca45d5368cd4c82af6153fbcc96a487ebd30b78b5f3c08718b7c07
+
+build:
+  skip: True  # [not linux]
+  number: 0
+
+requirements:
+  host:
+    - festival
+  run:
+    - festival
```

My understanding is that these festlex packages are just data packages and are thus the same on any platform. In that case, they should all be `noarch: generic` packages, and this section should be something like:

```suggestion
build:
  skip: True  # [not linux]
  number: 0
  noarch: generic

requirements:
  run:
  # add version constraints to festival package here
    - festival
```

---

Done, thanks.